docs/mkdocs.yml

```diff
@@ -22,69 +22,79 @@ plugins:
+      - metrics-reporting.md
+      - partitioning.md
+      - performance.md
+      - reliability.md
+      - schemas.md
+    - Views:
+      - view-configuration.md
+  - Libraries:
+    - Java:
+      - Quickstart: java-api-quickstart.md
+      - API: api.md
+      - Javadoc: ../../javadoc/latest/
+    - Python: https://py.iceberg.apache.org/
+    - Rust: https://rust.iceberg.apache.org/
+    - Go: https://pkg.go.dev/github.com/apache/iceberg-go/
   - Integrations:
-    - aws.md
-    - dell.md
+    # Iceberg-maintained integrations come first
+    - Apache Hive: hive.md
+    - Spark:
+      - spark-getting-started.md
+      - spark-configuration.md
+      - spark-ddl.md
+      - spark-procedures.md
+      - spark-queries.md
+      - spark-structured-streaming.md
+      - spark-writes.md
+    - Flink:
+      - flink.md
+      - flink-connector.md
+      - flink-ddl.md
+      - flink-queries.md
+      - flink-writes.md
+      - flink-actions.md
+      - flink-configuration.md
+    - Kafka Connect: kafka-connect.md
+    - Third-party:
+    # Now all the 3rd party integrations
+      - Apache Amoro: amoro.md
+      - Amazon Athena: https://docs.aws.amazon.com/athena/latest/ug/querying-iceberg.html
+      - Amazon Data Firehose: https://docs.aws.amazon.com/firehose/latest/dev/apache-iceberg-destination.html
+      - Amazon EMR: https://docs.aws.amazon.com/emr/latest/ReleaseGuide/emr-iceberg-use-cluster.html
+      - Amazon Redshift: https://docs.aws.amazon.com/redshift/latest/dg/querying-iceberg.html
+      - Apache Doris: https://doris.apache.org/docs/dev/lakehouse/catalogs/iceberg-catalog
+      - Apache Druid: https://druid.apache.org/docs/latest/development/extensions-contrib/iceberg/
+      - ClickHouse: https://clickhouse.com/docs/en/engines/table-engines/integrations/iceberg
+      - Daft: daft.md
+      - Databend: https://docs.databend.com/guides/access-data-lake/iceberg
+      - Dremio: https://docs.dremio.com/data-formats/apache-iceberg/
+      - DuckDB: https://duckdb.org/docs/preview/core_extensions/iceberg/overview
+      - Estuary: https://docs.estuary.dev/reference/Connectors/materialization-connectors/apache-iceberg/
+      - Firebolt: https://docs.firebolt.io/reference-sql/functions-reference/table-valued/read_iceberg
+      - Google BigQuery: https://cloud.google.com/bigquery/docs/iceberg-tables
+      - Impala: https://impala.apache.org/docs/build/html/topics/impala_iceberg.html
+      - Presto: https://prestodb.io/docs/current/connector/iceberg.html
+      - Redpanda: https://docs.redpanda.com/current/manage/iceberg/about-iceberg-topics
+      - RisingWave: risingwave.md
+      - Snowflake: https://docs.snowflake.com/en/user-guide/tables-iceberg
+      - Starrocks: https://docs.starrocks.io/en-us/latest/data_source/catalog/iceberg_catalog
+      - Tinybird: https://www.tinybird.co/docs/forward/get-data-in/table-functions/iceberg
+      - Trino: https://trino.io/docs/current/connector/iceberg.html
+  - Catalogs:
```

Comment:
This is not a complete list of catalogs (including community maintained and third-party). I prefer to put catalogs and storage into Integrations.

Reply:
But even HiveCatalog and HadoopCatalog are missing here.

Reply:
If they're not already listed, I'm not sure I see how this is adding any confusion?
Very happy to add as a fast-follow; I'll add them to the above issue.

Reply:
Catalogs are listed at https://iceberg.apache.org/terms/#catalog. If we want to add it to the Java docs, we'd better be consistent.

Reply:
> are you suggesting moving the Catalogs and Storage nested under Integrations? 

Yes, if they are not maintained by Iceberg community, I think they need to be under 3rd-party sub-section.